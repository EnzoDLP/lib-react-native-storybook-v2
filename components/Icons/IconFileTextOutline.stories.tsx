import type { Meta, StoryObj } from '@storybook/react';
import React from 'react';
import { View } from 'react-native';
import { IconFileTextOutline } from './IconFileTextOutline.component';

const meta = {
  title: 'Icons/IconFileTextOutline',
  component: IconFileTextOutline,
  decorators: [
    (Story) => (
      <View style={{ padding: 16 }}>
        <Story />
      </View>
    ),
  ],
} satisfies Meta<typeof IconFileTextOutline>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {
    accessibilityLabel: 'Ic√¥ne fichier',
    color: '#000000',
    size: 24,
  },
};
