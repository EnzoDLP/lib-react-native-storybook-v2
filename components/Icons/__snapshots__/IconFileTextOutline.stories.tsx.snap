// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons/IconFileTextOutline Default smoke-test 1`] = `
<div class="css-view-175oi2r"
     style="padding: 16px;"
>
  <svg width="24"
       height="24"
       viewbox="0 0 24 24"
       fill="none"
       aria-label="IcÃ´ne fichier"
       role="img"
  >
    <g clip-path="url(#clip0_41098_9886)">
      <path d="M17 14a1 1 0 01-1 1H8a1 1 0 110-2h8a1 1 0 011 1zm-4 3H8a1 1 0 000 2h5a1 1 0 100-2zm9-6.515V19a5.006 5.006 0 01-5 5H7a5.006 5.006 0 01-5-5V5a5.006 5.006 0 015-5h4.515a6.958 6.958 0 014.95 2.05l3.484 3.486A6.949 6.949 0 0122 10.485zm-6.949-7.021a5.008 5.008 0 00-1.05-.78V7a1 1 0 001 1h4.315a4.98 4.98 0 00-.78-1.05L15.05 3.464zm4.95 7.021c0-.165-.033-.323-.048-.485H15a3 3 0 01-3-3V2.047c-.162-.015-.32-.047-.485-.047H7a3 3 0 00-3 3v14a3 3 0 003 3h10a3 3 0 003-3v-8.515z"
            fill="#000000"
      >
      </path>
    </g>
  </svg>
</div>
`;
